crates/sncast/src/starknet_commands/script.rs

```diff
@@ -255,6 +322,10 @@ impl CairoHintProcessor<'_> {
                     .write(Felt252::from_(declare_response.transaction_hash))
                     .expect("Failed to insert transaction hash");
 
+                self.script_ui.print("");
```

Comment:
isn't there any other way? this looks a bit weird and redundant

Reply:
reworked